rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -43,3 +52,56 @@ impl Timer {
         LIVESPLIT_STATE.lock().unwrap().timer.set_game_time(TimeSpan::from_seconds(time));
     }
 }
+impl Run {
+    pub fn game_name() -> String {
+        LIVESPLIT_STATE.lock().unwrap().timer.run().clone().game_name().to_string()
+    }
+    pub fn set_game_name(game: Games) {
```

Comment:
Is there a use-case where you only set the game without also setting the category, src metadata (e.g. NGG), and the splits? Or are all of those always set at the same time? In the latter case, just have a single function to set all at the same time. That also makes it impossible to create an inconsistent state, e.g. having Maintained Max% NGG.